homeassistant/components/binary_sensor/zwave.py

```diff
@@ -50,13 +50,19 @@ class ZWaveBinarySensor(BinarySensorDevice, zwave.ZWaveDeviceEntity):
 
     def __init__(self, value, device_class):
         """Initialize the sensor."""
-        self._sensor_type = device_class
         zwave.ZWaveDeviceEntity.__init__(self, value, DOMAIN)
+        self._sensor_type = device_class
+        self._state = None
+        self.update_properties()
```

Comment:
This is done in all zwave components, and not doing it will result in no data until the device checks in. No problem with mains connected devices, but battery devices will not check in until the wakeup.
Can we accept this behaviour?